src/metabase/driver/postgres.clj

```diff
@@ -817,7 +817,9 @@
 
 (defmethod driver.sql/set-role-statement :postgres
   [_ role]
-  (format "SET ROLE %s;" role))
+  (if (= (u/upper-case-en role) "NONE")
+   (format "SET ROLE %s;" role)
+   (format "SET ROLE \"%s\";" role)))
```

Comment:
Can `role` contain `"` characters? Can it be escaped?

Reply:
Tested on both postgres and snowflake, can't seem to manually create a role that includes `"`, even by escaping them